### ISSUE TITLE
Exception thrown with keras output

### ISSUE THREAD

**Issue by josepablog** [hyperdashio/hyperdash-sdk-py#111]
I just spun off a new server, and unfortunately hyperdash is not working anymore.  I noticed that you released a new version, so I downgraded to 0.12.0, and that doesn't fix it : (

```
--> 270         self._dynamic_display = (sys.stdout.isatty() or
    271                                  'ipykernel' in sys.modules)
    272 

AttributeError: IOBuffer instance has no attribute 'isatty'
```

**Comment by richardartoul** [comment]
Hmm must be something new in Keras. I know how to fix it, I'll release a patched version tomorrow

**Comment by richardartoul** [comment]
Ok if you upgrade to version 0.13.1 you should be good to go. Fixed it in #112

**Comment by richardartoul** [comment]
Thanks for the report!
